checks/evaluation/pinned_dependencies.go

```diff
@@ -208,11 +209,20 @@ func addWorkflowPinnedResult(rr *checker.Dependency, w *worklowPinningResult, is
 	}
 }
 
-func createScoreForGitHubActionsWorkflow(wp *worklowPinningResult) []checker.ProportionalScoreWeighted {
+func logPinnedResult(dl checker.DetailLogger, p pinnedResult, name string) {
+	dl.Info(&checker.LogMessage{
+		Text: fmt.Sprintf("%3d out of %3d %s dependencies pinned", p.pinned, p.total, name),
```

Comment:
opinion/question: Is the `%3d` verb needed compared to just `%d`?

Reply:
This is fine for now, not sure how much it helps with readability but we can always remove in the future if needed.